experiences/homepage/homepage.test.js

```diff
@@ -52,6 +52,54 @@ const homepageTest = function() {
 		expect(cardExists).equal(true);
 	});
 
+	it('scroll bounded', async function() {
+		//TODO implement utility to get current SnapPoint
+		let currentSnapPoint = 623;
```

Comment:
Applying this change:
![image](https://user-images.githubusercontent.com/2574275/116881917-208f6280-ac24-11eb-8039-e2c7ff744ff9.png)

![image](https://user-images.githubusercontent.com/2574275/116881951-2a18ca80-ac24-11eb-9643-6bf8f0d6ee8f.png)


I get different values:
![image](https://user-images.githubusercontent.com/2574275/116881810-ff2e7680-ac23-11eb-98aa-fe7aaa895637.png)

Any clue why is that if the same code is executed (supposedly) and no scrolls in between are happening (supposedly)?